js/wazirx.js

```diff
@@ -0,0 +1,709 @@
+'use strict';
+
+const Exchange = require ('./base/Exchange');
+const { ExchangeError, BadRequest, DDoSProtection, RateLimitExceeded, BadSymbol, ArgumentsRequired, PermissionDenied, InsufficientFunds, InvalidOrder } = require ('./base/errors');
+
+module.exports = class wazirx extends Exchange {
+    describe () {
+        return this.deepExtend (super.describe (), {
+            'id': 'wazirx',
+            'name': 'WazirX',
+            'countries': ['IN'],
+            'version': 'v2',
+            'rateLimit': 100,
+            'has': {
+                'cancelAllOrders': true,
+                'cancelOrder': true,
+                'CORS': false,
+                'createOrder': true,
+                'fetchCurrencies': false,
+                'fetchBalance': true,
+                'fetchBidsAsks': false,
+                'fetchClosedOrders': false,
+                'fetchDepositAddress': false,
+                'fetchDeposits': true,
+                'fetchFundingFees': false,
+                'fetchFundingHistory': false,
+                'fetchFundingRate': false,
+                'fetchFundingRates': false,
+                'fetchMarkets': true,
+                'fetchMyTrades': false,
+                'fetchOHLCV': false,
+                'fetchOpenOrders': true,
+                'fetchOrder': true,
+                'fetchOrders': true,
+                'fetchOrderBook': true,
+                'fetchPositions': false,
+                'fetchStatus': false,
+                'fetchTicker': true,
+                'fetchTickers': true,
+                'fetchTime': true,
+                'fetchTrades': false,
+                'fetchTradingFee': false,
+                'fetchTradingFees': false,
+                'fetchTransactions': false,
+                'fetchWithdrawals': false,
+                'setLeverage': false,
+                'withdraw': false,
+                'fetchDepositAddressesByNetwork': false,
+                'transfer': false,
+                'fetchTransfers': false,
+            },
+            'urls': {
+                'logo': 'https://i0.wp.com/blog.wazirx.com/wp-content/uploads/2020/06/banner.png',
+                'api': {
+                    'spot': {
+                        'v1': 'https://api.wazirx.com/sapi/v1',
+                    },
+                },
+                'www': 'https://wazirx.com',
+                'doc': 'https://docs.wazirx.com/#public-rest-api-for-wazirx',
+            },
+            'api': {
+                'spot': {
+                    'v1': {
```

Comment:
Do they have any endpoints other than v1? If not, we can remove the `v1` subdivision from this dictionary, bringing it one layer up. Same as with `spot`.

Reply:
@kroitor same as above, but I can lift it 🙂